.github/workflows/check-expo-dependencies.yml

```diff
@@ -0,0 +1,45 @@
+name: check-expo-dependencies
+on:
+  workflow_call:
+    outputs:
+      cancel-workflow:
+        description: "Return 'true' if PR was closed and the workflow needs to be cancelled"
+        value: ${{ jobs.check.outputs.cancel-workflow }}
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    outputs:
+      cancel-workflow: ${{ steps.should-cancel-workflow.outputs.result }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: .nvmrc
+      - run: yarn
```

Comment:
why do we need to check out the source code and install packages again?

Reply:
try moving the usage of this workflow **after** the checkout action in the calling workflow :)